library/src/containers/Sidebar/Sidebar.tsx

```diff
@@ -75,6 +79,7 @@ export const Sidebar: React.FunctionComponent = () => {
 
   const operationList = hasOperations && (
     <>
+      <h1>Testing</h1>
```

Comment:
I don't think it should stay in the code

Reply:
Oh! 🫣